addons/mail/static/src/core/common/message.xml

```diff
@@ -45,15 +45,15 @@
                                 className="'ms-1'"
                                 message="props.message"
                                 thread="props.thread"/>
-                            <small t-if="!isOriginThread and !message.is_transient" t-on-click.prevent="openRecord" class="ms-1 text-500">
+                            <small t-if="isNotOriginThreadAndIsNotTransient" t-on-click.prevent="openRecord" class="ms-1 text-500">
                                 <t t-if="message.model !== 'discuss.channel'">
                                     on <a t-att-href="message.resUrl"><t t-esc="message.originThread?.displayName"/></a>
                                 </t>
                                 <t t-if="message.model === 'discuss.channel'">
                                     (from <a t-att-href="message.resUrl"><t t-esc="message.originThread?.prefix"/><t t-esc="message.originThread?.displayName"/></a>)
                                 </t>
                             </small>
-                            <div t-if="props.message.scheduledDate" t-att-class="{ 'ms-2': props.isInChatWindow and isAlignedRight }" t-att-title="messageService.scheduledDateSimple(props.message)">
+                            <div t-if="props.message.scheduledDate" t-att-class="{ 'ms-2': (props.isInChatWindow and isAlignedRight) or (isNotOriginThreadAndIsNotTransient)}" t-att-title="messageService.scheduledDateSimple(props.message)">
```

Comment:
Nitpick: keep whitespace before `}`, since we have one after `{`. The whitespace are preferable for readability